web-client/src/presenter/actions/navigation/toggleMenuStateAction.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line spellcheck/spell-checker
```

Comment:
might be better to just change `DropdownMenu` => `DropDownMenu`. And hyphenate unsetting if it still complains

Reply:
Updated!